sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -529,69 +603,178 @@ myparams::joint_matrix_b<sub_group, layout::row_major> sub_b;
+each of these members has the value zero
+|`atype`, `btype`, `accumulatortype`| indicates the types supported in
+the combination. these are of type `matrix_type` which tells the list
+of types that are supported for the A, B, and accumulator matrices in
+the `T` template parameter as follows: +
+`bf16`: `sycl::bfloat16` +
+`fp16`: `sycl::half` +
+`tf32`: `sycl::ext::oneapi::experimental::matrix::precision::tf32` +
+`fp32`: `float` +
+`fp64`: `double` +
+`sint8`: signed 8 bits signed integer +
+`sint16`: `signed short` +
+`sint32`: `signed int` +
+`sint64`: `signed long` +
+`uint8`: unsigned 8 bits integer +
+`uint16`: `unsigned short` +
+`uint32`: `unsigned int` +
+`uint64`: `unsigned long` 
+|======================
+
+===== Runtime Query Example:
 ```c++
-auto data = get_wi_data(sg, C);
-for (int i = 0; i < data.length; ++i) {
-  auto row, col = data[i].get_coord();
-  sum_of_local_rows[row] += data[i];
+// Ta, Tb, Taccumulator are the types used in applications
+std::vector<combination> combinations =
+           device.get_info<info::device::matrix::combinations>();
+for (int i = 0; sizeof(combinations); i++) {
+  if (Ta == combinations[i].atype &&
+      Tb == combinations[i].btype &&
+      Tc == combinations[i].accumulatortype) {
+    // joint matrix GEMM kernel can be called using these sizes
+    joint_matrix_gemm(combinations[i].msize,
+         combinations[i].nsize, combinations[i].ksize);
+  }
 }
 ```
 
-## TODO List
-- Add WI data to joint matrix mapping coordinates information for piece-wise operations. This will be added as part of the query or new methods to the 'get_wi_data' class. 
-- Add a more realistic and complete example that shows the value of the general query. 
+=== Appendix: Supported Combinations Per Hardware
+
+The table below provides a list of the combinations that
+`joint_matrix` implementations support on each of Intel AMX and Intel
+XMX hardware. Note that these can be returned in a parametrized way
+using the `matrix_params` query class.
+
+==== Intel AMX Supported Combinations
+This is currently available in
+`sycl::ext::oneapi::experimental::architecture::intel_cpu_spr`.
+
+[frame="none",options="header"]
+|======================
+| A type | B type | Accumulator type | M | N | K
+| `matrix_type::(u)int8`  | `matrix_type::(u)int8` |
+`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
+|  `matrix_type::bf16`       |  `matrix_type::bf16`   |
+`matrix_type::fp32`   |  +<=+ 16 |  +<=+ 16   |  +<=+ 32
+|======================
+
+==== Intel XMX Supported Combinations
+This is currently available in
+`sycl::ext::oneapi::experimental::architecture::intel_gpu_pvc` and
+`sycl::ext::oneapi::experimental::architecture::intel_gpu_dg2`.
+
+[frame="none",options="header"]
+|======================
+| A type | B type | Accumulator type | M | N | K | device
+| `matrix_type::(u)int8`  | `matrix_type::(u)int8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 |
+sycl::ext::oneapi::experimental::architecture::intel_gpu_pvc
```

Comment:
```suggestion
architecture::intel_gpu_pvc
```

The table is hard to read with these long names.  Shortening it like this will make it easier to read.